examples/pytorch/pinsage/README.md

```diff
@@ -28,4 +28,16 @@ item embeddings, which are learned as outputs of PinSAGE.
 python model.py data.pkl --num-epochs 300 --num-workers 2 --device cuda:0 --hidden-dims 64
 ```
 
+The implementation here also assigns a learnable vector to each item.  If your hidden
+state size is so large that the learnable vectors cannot fit into GPU, use this script
+for sparse embedding update (written with `torch.optim.SparseAdam`) instead:
+
+
+```
+python model_sparse.py data.pkl --num-epochs 300 --num-workers 2 --device cuda:0 --hidden-dims 1024
```

Comment:
Does sparse embedding increase the speed?

Reply:
Also, maybe it's better to list the performance of it.

Reply:
> Does sparse embedding increase the speed?

No it decreases the speed.